### DISCUSSION TITLE
Mouse coords

### DISCUSSION THREAD

**Discussion by jumpjack** [ricktu288/ray-optics#134] [Ideas]
https://github.com/ricktu288/ray-optics/blob/76b103e47f4e9680b17fd7291e96b92bd695938a/simulator/js/editor.js#L272

Add here  this line to constantly show mouse coords somewhere:

myStatus.innerHTML = "X=" + mouse.x;+ + ", Y=" + mouse.y;

**Comment by ricktu288** [comment]
You can add that text to the status box at the lower left corner (which can be enabled by Settings -> Show status box. Then you can open a PR.

**Comment by jumpjack** [comment]
I tried editing index.html but I am not able, I only know plain HTML/Javascript, you used some kind of framework which I don't know.

**Comment by ricktu288** [comment]
Added.
